timelock-server/versions.lock

```diff
@@ -1336,7 +1334,7 @@
             ]
         },
         "com.fasterxml.jackson.module:jackson-module-jaxb-annotations": {
-            "locked": "2.7.8",
+            "locked": "2.6.7",
```

Comment:
Hmm, good spot. I'll switch back to forcing 2.7.8 for jaxb-annotations for now, as I don't quite understand how/if this might affect things.

TimeLock Jackson dependency versions are rather messy, because of the different version of Dropwizard we're using compared to the atlasdb dropwizard bundle / mandated by e.g. atlasdb-config. I've filed #1800 for this.